docs/topics/proc_configuring-your-application-to-use-spring-boot.adoc

```diff
@@ -1,16 +1,15 @@
 [id='configuring-your-application-to-use-spring-boot_{context}']
 = Configuring your application to use {SpringBoot}
 
-To configure your application to use {SpringBoot}:
+Reference the {SpringBoot} BOM (Bill of Materials) artifact in the `pom.xml` file at the root directory of your application.
 
 .Prerequisites
 
 * A Maven-based application
 
 .Procedure
 
-. Reference the {SpringBoot} BOM (Bill of Materials) artifact in the `<dependencyManagement>` section of the `pom.xml` file of your application.
-Specify the `<type>pom</type>` and `<scope>import</scope>`:
+. Open the `pom.xml` file and add the `<dependencyManagement>` artifact  and specify the `<type>pom</type>` and `<scope>import</scope>`:
```

Comment:
@huntley I'm still not sure whether the formulation _`<dependencyManagement>` artifact_ is semantically correct. Your opinion, @spisiakm?

Reply:
It's not. Again, see https://github.com/fabric8-launcher/launcher-documentation/pull/1239#discussion_r246795747 . `<dependencyManagement>` is not an artifact, it's a section in a `pom.xml` file that does some specific stuff.